main/src/org/destinationsol/game/chunk/ChunkFiller.java

```diff
@@ -247,9 +247,9 @@ private void fillFarJunk(SolGame game, Vector2 chCenter, RemoveController remove
         TextureManager textureManager = game.getTexMan();
 
         for (int i = 0; i < count; i++) {
-            // Select a random far junk texture
+            // Select a random far junk atlas
```

Comment:
Oh damn, this was a refactor thing, not intentional. Definitely should be junk texture.
Sorry! :P